### ISSUE TITLE
Consider creating a separate "setup" repository for reuse in lessons

### ISSUE THREAD

**Issue by karlbenedict** [datacarpentry/geospatial-workshop#40]
The current collection of geospatial workshop lessons have common setup requirements that have been duplicated into the separate lesson repositories. The issues and associated pull requests that I recently created (#35, #36, #37, #38, #39) apply to these multiple copies of `setup.md` in these separate repositories. It might be a good practice to set up a separate `setup` repository that contains the shared setup content and either point to its content or include it as a git submodule in the separate repositories that reuse the material. This would streamline management and updating of the setup code in the long run.

**Comment by chris-prener** [comment]
Agreed that this is a bit confusing... an alternative might be to remove the setup instructions entirely from the various repos and link back to the main set of setup instructions.
